var/spack/repos/builtin/packages/py-pycompadre/package.py

```diff
@@ -35,19 +38,30 @@ class PyPycompadre(PythonPackage):
         description="Debugging level 0) release 1) debug 2) extreme debugging",
     )
 
-    depends_on("cmake@3.10.0:", type="build")
-    depends_on("python@3.4:", type=("build", "link", "run"))
-    depends_on("py-pip", type=("build", "link", "run"))
+    depends_on("cmake@3.13:", type="build", when="@:1.4")
+    depends_on("cmake@3.16:", type="build", when="@1.5:")
+
+    depends_on("kokkos-kernels@3.3.01:4", when="@:1.5")
+    depends_on("kokkos-kernels@4:", when="@1.6:")
+
+    depends_on("python@3.6:", type=("build", "link", "run"))
```

Comment:
Thank you for your looking at this carefully. `numpy` is required according to setup.py, but it is not a build requirement (it is used in testing). This will be cleaned up in a future modification of setup.py, but is not a dependency we would like to register in Spack.